assignments/prototypes.js

```diff
@@ -14,97 +13,173 @@
   * dimensions
   * destroy() // prototype method -> returns the string 'Object was removed from the game.'
 */
+const GameObject = function(obj) {
+  this.createdAt = obj.createdAt;
+  this.dimensions = obj.dimensions;
+};
+GameObject.prototype.destroy = function() {
+  return `${this.name} was removed from the game.`;
+};
 
 /*
-  === CharacterStats ===
-  * hp
-  * name
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
+=== CharacterStats ===
+* hp
+* name
+* takeDamage() // prototype method -> returns the string '<object name> took damage.'
+* should inherit destroy() from GameObject's prototype
 */
 
+const CharacterStats = function(obj) {
+  GameObject.call(this, obj);
+  this.hp = obj.hp;
+  this.name = obj.name;
+};
+// console.log(GameObject.prototype);
+CharacterStats.prototype = {...GameObject.prototype};
+CharacterStats.prototype.takeDamage = function() {
+  return `${this.name} took damage.`;
+};
+
+
 /*
-  === Humanoid ===
-  * faction
-  * weapons
-  * language
-  * greet() // prototype method -> returns the string '<object name> offers a greeting in <object language>.'
-  * should inherit destroy() from GameObject through CharacterStats
-  * should inherit takeDamage() from CharacterStats
+=== Humanoid ===
+* faction
+* weapons
+* language
+* greet() // prototype method -> returns the string '<object name> offers a greeting in <object language>.'
+* should inherit destroy() from GameObject through CharacterStats
+* should inherit takeDamage() from CharacterStats
 */
- 
+const Humanoid = function(obj) { 
+ CharacterStats.call(this, obj);
+  this.faction = obj.faction;
+  this.weapons = obj.weapons;
+  this.language = obj.language;
+};
+
+//Overwrite my prototype with {this stuff, which is CharacterStats}
+Humanoid.prototype = {...CharacterStats.prototype};
+
+Humanoid.prototype.greet = function() {
+ return `${this.name} offers a greeting in ${this.language}.`;
+}
+
+Humanoid.prototype.poke = function(Hero) {
+  return `${this.name} poked ${} for 7 damage.`;
```

Comment:
Error: in this line nothing is passed in inside ${ } after the word poked. I suggest passing Hero inside those curly braces.